moveit_ros/moveit_servo/include/moveit_servo/status_codes.hpp

```diff
@@ -48,22 +50,22 @@ enum class StatusCode : int8_t
   INVALID = -1,
   NO_WARNING = 0,
   DECELERATE_FOR_APPROACHING_SINGULARITY = 1,
-  HALT_FOR_SINGULARITY = 2,
-  DECELERATE_FOR_COLLISION = 3,
-  HALT_FOR_COLLISION = 4,
-  JOINT_BOUND = 5,
-  DECELERATE_FOR_LEAVING_SINGULARITY = 6,
-  PAUSED = 7
+  HALT_FOR_SINGULARITY = 3,
+  DECELERATE_FOR_LEAVING_SINGULARITY = 4,
+  DECELERATE_FOR_COLLISION = 5,
+  HALT_FOR_COLLISION = 6,
+  JOINT_BOUND = 7,
+  POSE_ACHIEVED = 8
```

Comment:
Did you skip 2 on accident?  ;)

```suggestion
  HALT_FOR_SINGULARITY = 2,
  DECELERATE_FOR_LEAVING_SINGULARITY = 3,
  DECELERATE_FOR_COLLISION = 4,
  HALT_FOR_COLLISION = 5,
  JOINT_BOUND = 6,
  POSE_ACHIEVED = 7
```

Reply:
I think had added another status which I assigned 2, forgot to adjust the numbering after I removed it. Thanks